cmd/buildah/rmi.go

```diff
@@ -33,11 +38,16 @@ var (
 
 func rmiCmd(c *cli.Context) error {
 	force := c.Bool("force")
+	removeAll := c.Bool("all")
 
 	args := c.Args()
-	if len(args) == 0 {
+	if len(args) == 0 && !c.Bool("all") {
```

Comment:
why not using `removeAll` instead of `c.Bool("all")`

Reply:
I should, thx for the ptr!